npm/private/pnpm.bzl

```diff
@@ -531,12 +531,12 @@ def _parse_lockfile(parsed, err):
         A tuple of (importers dict, packages dict, patched_dependencies dict, error string)
     """
     if err != None or parsed == None or parsed == {}:
-        return {}, {}, {}, err
+        return {}, {}, {}, -1, err if err != None else "could not parse Yaml config"
```

Comment:
I'm confused, you now return a 5-tuple instead of a 4-tuple, doesn't that require changes at the callsites?

Reply:
The call sites already expect 5-tuple 😄 — they never see the invalid 4-tuple though because of no errors most of the time

Reply:
I see, do you mind correcting the `Returns` doc above in that case?

Reply:
This change looks correct, where the `-1` is the `lockfileVersion`.

However I don't think we should have the default `"could not parse Yaml config"` since I think an empty lockfile is valid? The `-1` version number is probably still ok in that case.

Can we add a test for that case that failed previously?